certbot/src/certbot/_internal/renewal.py

```diff
@@ -328,6 +329,27 @@ def should_renew(config: configuration.NamespaceConfig,
     display_util.notify("Certificate not yet due for renewal")
     return False
 
+
+def _default_renewal_time(cert_pem: bytes) -> datetime.datetime:
```

Comment:
I could also put this in acme utils if we really want others to have it, but since it doesn't require any external calls it really does not belong in acme client

Reply:
this makes sense to me

i think putting it here is the easiest and sanest thing for us, but if anyone on the team really wants it in acme, i'd be fine with that too

Reply:
I agree. This should be internal until / unless we hear from someone who wants to use it.